packages/workspace/src/browser/workspace-input-dialog.ts

```diff
@@ -54,4 +57,36 @@ export class WorkspaceInputDialog extends SingleTextInputDialog {
         // Add the path and icon div before the `inputField`.
         this.contentNode.insertBefore(element, this.inputField);
     }
+
+    /**
+     * Append the human-readable absolute root `path` to the dialog.
+     */
+    protected appendRootName(): void {
+        if (this.workspaceService && this.workspaceService.isMultiRootWorkspaceOpened) {
+            const roots = this.workspaceService.tryGetRoots();
+            const RootsWithSameName = roots.filter(singleRoot => {
+                const singleRootUri = new URI(singleRoot.uri);
+                return this.labelProvider.getName(this.props.parentUri) === this.labelProvider.getName(singleRootUri);
+            });
+            const root = this.workspaceService.getWorkspaceRootUri(this.props.parentUri);
```

Comment:
A more descriptive name would be `rootUri` due to the return type of the method.